avro-fastserde/src/main/java/com/linkedin/avro/fastserde/FastSerdeCache.java

```diff
@@ -296,6 +329,24 @@ private String getSchemaKey(Schema writerSchema, Schema readerSchema) {
         Utils.getSchemaFingerprint(readerSchema));
   }
 
+  /**
+   * This function can get the total schema number limit for {@link FastSerdeCache}
+   *
+   * @return current schema number limit
+   */
+  public int getSchemaNumLimit() {
+    return schemaNumLimit;
+  }
+
+  /**
+   * Set the total schema number limit for {@link FastSerdeCache}.
+   * It is the total limit for fast Generic/Specific De/Serializer
+   * If the limit is set for multiple times, the last limit will be used
+   */
+  public void setSchemaNumLimit(int limit) {
```

Comment:
I think it is better to pass the `limit` as the param of a new constructor and it shouldn't change during the lifetime of `FastSerdeCache`.
If we choose to expose a `setter` here, when the user specifies a lower number after generating a lot of fast classes, fast-avro won't unload the loaded classes to satisfy the new limit, which is a little confusing.

And I don't see a reason people would change the limit dynamically since they have no notion about how many classes have been generated or how many more are needed in the future when the application is running.

Reply:
1. schemaNumLimit is static - so this method should be static as well.
2. that means there's no constructor to set this in?
3. i would expect this would be called once, during application boot. however i see no damage is changing this at runtime